lib/yookassa/payments.rb

```diff
@@ -14,6 +17,8 @@ def find(payment_id:)
     end
 
     def create(payment:, idempotency_key: SecureRandom.hex(10))
+      validate_payment!(payment)
```

Comment:
In ruby world (not rails) methods with bang sign usually mean that current method mutate its state.